source/libuv/uv_event_loop.c

```diff
@@ -216,7 +216,7 @@ static void s_do_subscribe(struct libuv_loop *impl, struct handle_data *handle_d
     handle_data->uv_events |= UV_DISCONNECT;
 #endif
 
-    /* Start listening for events */
+    /* Start listening for events. The callback should never actually be called, so we use a poison. */
```

Comment:
Could we not just route it to a callback of our own that asserts fatally? At least then we'd understand the callstack?